app/routes/register-your-interest/register-your-interest.js

```diff
@@ -8,7 +9,11 @@ module.exports = [
     options: {
       auth: false,
       handler: async (request, h) => {
-        return h.view('register-your-interest/register-your-interest', { ruralPaymentsAgency })
+        if (defraIdConfig.enabled) {
+          return h.view('defra-id/register-your-interest/register-your-interest', { ruralPaymentsAgency })
```

Comment:
could probably remove the duplicate registration of interest and have this as an index page?